app/src/main/java/org/simple/clinic/patient/recent/RecentPatientsView.kt

```diff
@@ -71,21 +76,33 @@ class RecentPatientsView(context: Context, attrs: AttributeSet) : FrameLayout(co
         .ofType<RecentPatientItemClicked>()
```

Comment:
Can we not handle clicks like this in the screen separately? This means that the events never get reported to Heap since this is done in the controller. The `adapterEvents` should be fed into the stream that goes to the controller and these events should be handled in the controller.

Reply:
done